### PR TITLE
rename useImperativeMethods -> useImperativeHandle

### PR DESCRIPTION
**Pull request by threepointone**
A followup from https://github.com/facebook/react/pull/14565; don't merge this till we land that one. thanks!

### CODE REVIEW

File: content/docs/hooks-reference.md [hunk 1]
```diff
@@ -321,18 +321,18 @@ function TextInputWithFocusButton() {
 
 Note that `useRef()` is useful for more than the `ref` attribute. It's [handy for keeping any mutable value around](/docs/hooks-faq.html#is-there-something-like-instance-variables) similar to how you'd use instance fields in classes.
 
-### `useImperativeMethods`
+### `useImperativeHandle`
 
 ```js
-useImperativeMethods(ref, createInstance, [inputs])
+useImperativeHandle(ref, createInstance, [inputs])
```
Comment (gaearon) [inline content/docs/hooks-reference.md]: Maybe `createHandle`

File: content/docs/hooks-reference.md [hunk 2]
```diff
@@ -321,18 +321,18 @@ function TextInputWithFocusButton() {
...
 
-`useImperativeMethods` customizes the instance value that is exposed to parent components when using `ref`. As always, imperative code using refs should be avoided in most cases. `useImperativeMethods` should be used with `forwardRef`:
+`useImperativeHandle` customizes the instance value that is exposed to parent components when using `ref`. As always, imperative code using refs should be avoided in most cases. `useImperativeHandle` should be used with `forwardRef`:
 
 ```js
 function FancyInput(props, ref) {
   const inputRef = useRef();
-  useImperativeMethods(ref, () => ({
+  useImperativeHandle(ref, () => ({
     focus: () => {
```
Comment (gaearon) [inline content/docs/hooks-reference.md]: Maybe we should change this to promote the static call style. You’ll need to discuss this with Seb.

Comment (threepointone) [reply content/docs/hooks-reference.md]: static call style? like, `React.` ? I'll reach out.

Comment (gaearon) [reply content/docs/hooks-reference.md]: No, I mean not using an object at all. Although I guess in that case `useImperativeHandle` would be unnecessary in the first place. Disregard.

**Comment by bvaughn** [comment]
Since I just published a new alpha, I'm rolling out this change also

**Pull request merged**